### DISCUSSION TITLE
Matter end device unresponsive

### DISCUSSION THREAD

**Discussion by djandrew2005** [openthread/openthread#9034] [Q&A]
I am using Google Nest Hub and I have connected to it a matter-over-thread device. The device works with Google Home just fine but when I try to pair with other platforms the device becomes unresponsive on the other platforms but still working in Google Home.

I have tried adding it to Home Assistant from an Android device (Thread Credentials did transfer to Home Assistant while adding) and to Apple Home with iOS 16.5 (they added support in this version after my request for third party Thread border routers)

I know this sounds like a matter problem but I am pretty sure that it is not.

**Comment by jwhui** [comment]
Can you provide any supporting information that leads you to believe that this is an OpenThread issue?

The fact that it is still working with Google Home indicates that Thread communication is still working.

**Reply by djandrew2005** [reply]
Sorry I don't have more info. From my understanding other ecosystems can't access the thread network. They told me that there is probably something to do with mdns
